bin/openshift-operations

```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "http"
+  config.host   = "#{ENV["SOURCES_HOST"]}:#{ENV["SOURCES_HOST"]}"
```

Comment:
Think this should be `"#{ENV["SOURCES_HOST"]}:#{ENV["SOURCES_PORT"]}"`

Reply:
:+1: derp, done